internal/tiltfile/tiltfile_test.go

```diff
@@ -536,56 +536,6 @@ func TestReadFile(t *testing.T) {
 	assert.Equal(t, manifest.K8sYaml, "hello world")
 }
 
-func TestConfigMatcherWithFastBuild(t *testing.T) {
```

Comment:
let's keep this test, but modify it to create the Matcher from ConfigFiles. this should be trivial because there's already a method for it, come talk to me if it's not trivial